Portal/src/Datahub.Portal/Pages/ShareSpatial.razor

```diff
@@ -32,7 +33,7 @@
             {
                 <AeCard>
                     <AeTypography Variant="h3">@StepTitle(SpatialObjectShareStatus.FillApprovalForm)</AeTypography>
```

Comment:
Just a heads up there's some Ae components here, can remove later

Reply:
I believe this is an unused legacy component - likely why it hadn't been updated with MudBlazor in the first place.